### ISSUE TITLE
Align number cols / cells to the right?

### ISSUE THREAD

**Issue by PulsarFX** [Plasma/csv-to-table#14]
Would it be possible, to make cells or columns with numbers  right aligned so they are easier to read?

In my case it would turn this:
```
|--------------|-----------|-----------|
| 2021         | eNiro     | Verbrauch |
|--------------|-----------|-----------|
| Fahrleistung | 19.600,00 | km        |
|--------------|-----------|-----------|
| Stromkosten  | 1.132,00  | €         |
|--------------|-----------|-----------|
| Strommenge   | 3.496,00  | kWh       |
|--------------|-----------|-----------|
| Verbrauch Ø  | 17,84     | kWh/100km |
|--------------|-----------|-----------|
| Preis Ø      | 0,32      | €/kWh     |
|--------------|-----------|-----------|
| Strom pro km | 0,06      | €         |
|--------------|-----------|-----------|
```

to this:
```
|--------------|-----------|-----------|
|         2021 | eNiro     | Verbrauch |
|--------------|-----------|-----------|
| Fahrleistung | 19.600,00 | km        |
|--------------|-----------|-----------|
| Stromkosten  |  1.132,00 | €         |
|--------------|-----------|-----------|
| Strommenge   |  3.496,00 | kWh       |
|--------------|-----------|-----------|
| Verbrauch Ø  |     17,84 | kWh/100km |
|--------------|-----------|-----------|
| Preis Ø      |      0,32 | €/kWh     |
|--------------|-----------|-----------|
| Strom pro km |      0,06 | €         |
|--------------|-----------|-----------|

```

**Comment by Plasma** [comment]
Thanks for the suggestion, I'll look to implement this in an upcoming release soon.

**Comment by Plasma** [comment]
This feature is now available in the latest `1.4.0` release and enabled by default, you can turn it off via extension settings.
